frontend/src/components/User/User.js

```diff
@@ -96,7 +76,8 @@ export const User = props => {
 
         isUserFollowing();
         loadUser();
-      }, [props.username, isFollowing])
+
+    }, [props.username, isFollowing])
 
```

Comment:
What changed here? Was an extra blank line added? If that's it I think it should be removed.

Reply:
Yeah, I think it was a blank line to isolade `loadUser();` visually